junit-vintage-engine/src/main/java/org/junit/vintage/engine/descriptor/RunnerTestDescriptor.java

```diff
@@ -161,6 +171,10 @@ public boolean isIgnored() {
 		return ignored;
 	}
 
+	public void setScheduler(RunnerScheduler scheduler) {
+		this.scheduler = scheduler;
+	}
```

Comment:
> I want to design it similarly to JUnit 4's ParallelComputer, but I need help..🥹
> 
>     * [junit-team/junit4@`df00d5e`/src/main/java/org/junit/experimental/ParallelComputer.java#L54-L66](https://github.com/junit-team/junit4/blob/df00d5eced3a7737b88de0f6f9e3673f0cf88f88/src/main/java/org/junit/experimental/ParallelComputer.java#L54-L66)

Wouldn't this suffice?

```suggestion
	public void setScheduler(org.junit.runners.model.RunnerScheduler scheduler) {
		Runner runner = getRunnerToReport(); 
		if (runner instanceof ParentRunner) {
			((ParentRunner<?>) runner).setScheduler(scheduler);
		}
	}
```